lib/ord.js

```diff
@@ -29,8 +29,7 @@ const fInitializeGlobal = (csn) => {
     const aServices = [];
     const aODMEntity = [];
 
-    const capNamespace = csn.namespace;
-     // namespace variable value if present in cdsrc.json take it there or else from package.json
```

Comment:
Not needed if we use `global.namespace`(value given in .cdsrc.json || package.json). 

@Fannon @zongqichen : Do you agree?